### PR TITLE
[INFRA-1388] Add a bigger k8s cluster

### PR DESCRIPTION
**Pull request by olblak**

### CODE REVIEW

File: plans/bean.tf [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    admin_username = "azureuser"
+
+    ssh_key {
+      key_data = "${file("${var.ssh_pubkey_path}")}"
+    }
+  }
+
+  agent_pool_profile {
+    name       = "bean"
+    count      = 3
```
Comment (rtyler) [inline plans/bean.tf]: I think this should be 5! MOAR COMPUTE!

File: plans/bean.tf [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    dns_prefix = "agent${azurerm_resource_group.bean.name}"
+    vm_size    = "Standard_D8_v3"
```
Comment (rtyler) [inline plans/bean.tf]: :shipit:

Comment (olblak) [reply plans/bean.tf]: 'Standard_D8_v3' is not allowed so either we upgrade terraform to the latest version either we use one of the following vm size.
Anyway I am looking to upgrade  Terraform from 0.9 to 0.10 

The value 'Standard_D8_v3' of parameter 'agentProfile.vmSize' is not allowed. Allowed values are 'Standard_A0, Standard_A1, Standard_A2, Standard_A3, Standard_A4, Standard_A5, Standard_A6, Standard_A7, Standard_A8, Standard_A9, Standard_A10, Standard_A11, Standard_D1, Standard_D2, Standard_D3, Standard_D4, Standard_D11, Standard_D12, Standard_D13, Standard_D14, Standard_D1_v2, Standard_D2_v2, Standard_D2_v2_Promo, Standard_D3_v2, Standard_D3_v2_Promo, Standard_D4_v2, Standard_D4_v2_Promo, Standard_D5_v2, Standard_D5_v2_Promo, Standard_D11_v2_Promo, Standard_DS11_v2, Standard_D12_v2, Standard_D12_v2_Promo, Standard_D13_v2, Standard_D13_v2_Promo, Standard_D14_v2, Standard_D14_v2_Promo, Standard_D15_v2, Standard_G1, Standard_G2, Standard_G3, Standard_G4, Standard_G5, Standard_A1_v2, Standard_A2_v2, Standard_A2m_v2, Standard_A4_v2, Standard_A4m_v2, Standard_A8_v2, Standard_A8m_v2, Standard_D15_v2, Standard_F1, Standard_F16, Standard_F2, Standard_F4, Standard_F8, Standard_H16, Standard_H16m, Standard_H16mr, Standard_H16r, Standard_H8, Standard_H8m, Standard_L16s, Standard_L32s, Standard_L4s, Standard_L8s, Standard_DS1, Standard_DS2, Standard_DS3, Standard_DS4, Standard_DS11, Standard_DS12, Standard_DS13, Standard_DS14, Standard_GS1, Standard_GS2, Standard_GS3, Standard_GS4, Standard_GS5, Standard_DS1_v2, Standard_DS2_v2, Standard_DS3_v2, Standard_DS4_v2, Standard_DS5_v2, Standard_DS11_v2, Standard_DS12_v2, Standard_DS13_v2, Standard_DS14_v2, Standard_DS15_v2, Standard_DS2_v2_Promo, Standard_DS3_v2_Promo, Standard_DS4_v2_Promo, Standard_DS5_v2_Promo, Standard_F16s, Standard_F1s, Standard_F2s, Standard_F4s, Standard_F8s, Standard_NC12, Standard_NC24, Standard_NC24r, Standard_NC6, Standard_NV12, Standard_NV24, Standard_NV6'."

Comment (olblak) [reply plans/bean.tf]: So definitely Standard_D8_V3 is not available for ACS therefor I suggest to use Standard_DS4_V2 which is the equivalent.

**Pull request merged**